translations/base-de.yaml

```diff
@@ -677,30 +677,30 @@ settings:
         soundsMuted:
             title: Geräusche stummschalten
             description: >-
-                Bei Aktivierung werden alle Geräusche stummgeschaltet.
+                Schaltet alle Geräusche stumm.
 
         musicMuted:
             title: Musik stummschalten
             description: >-
-                Bei Aktivierung wird die Musik stummgeschaltet.
+                Schaltet die Musik stumm.
 
         theme:
             title: Farbmodus
             description: >-
-                Wähle zwischen dunklem und hellem Farbmodus.
+                Wähle zwischen dem dunklen und dem hellen Farbmodus.
 
             themes:
                 dark: Dunkel
                 light: Hell
         refreshRate:
-            title: Zielbildwiederholrate
+            title: Bildwiederholungsrate
```

Comment:
`Simulation Target` ist das zu übersetzende Wort (du lässt `Target` unter den Tisch fallen). `Bildwiederholrate` ist geläufiger (und leichter auszusprechen) als `Bildwiederholungsrate`.

Reply:
Habe `-ungs-` entfernt. In welchem Kontext ist da nun `Simulation Target`???

Reply:
`Simulation Target` ist der Begriff der in der Originalsprache verwendet wird (s. `base-en.yaml`). Das ist praktisch der Begriff, der von uns übersetzt werden soll.

Reply:
Ich glaube es wurde aber kürzlich in `tick rate` umbenannt. Da müssen wir wohl eine neue Übersetzung finden

Reply:
Wenn ich es richtig verstanden habe werden die FPS gemeint, dann würde ich `Bildwiederholrate` entweder stehen lassen, oder in FPS umwandeln. Auch wenn FPS eigentlich englisch ist, wird es meines Wissens nach auch im deutschen so gebraucht.
Wenn nicht die FPS gemeint sind, weiß ich nicht was ansonsten gemeint ist.

Reply:
Hier ist tatsächlich die `Tick Rate` (vgl. neuere Version) gemeint:
![grafik](https://user-images.githubusercontent.com/57270769/90341671-bedd8880-e001-11ea-8f6d-d8f0dbbcb38a.png)
Ich würde sagen wir übersetzen einfach die neue Version. `Tick Rate` ist, wie viele Simulationsschritte pro Sekunde berechnet werden. Also wie 'smooth' die Simulation ist. Wird auch einfach mit `Tickrate` übersetzt. Kannst du den Originaltext suchen und eine neue Übersetzung finden? Ansonsten kann ich das auch machen, in ca. 30 Min.

Reply:
ich schaue mal nach

Reply:
Ich habe jetzt die Englische Version übersetzt.